File: TIENDARENEGUL/src/Belleza.java
import java.util.Date;
// Clase Belleza que extiende de Producto
class Belleza extends Producto {
    // Atributos y métodos específicos de Belleza
    private String esOrganico;

    // Constructor
    public Belleza(String nombre, double precio, String fechaImportacion, String numeroSerie, String esOrganico, int stock) {
        super(nombre, precio, fechaImportacion, numeroSerie, stock);
        this.esOrganico = esOrganico;
    }

    // Getter y Setter específicos
    public String getEsOrganico() {
        return esOrganico;
    }

    public void setEsOrganico(String esOrganico) {
        this.esOrganico = esOrganico;
    }

}

File: TIENDARENEGULPY/Belleza.py
from Belleza import Producto  # Suponiendo que Producto es una clase existente en otro archivo llamado producto.py

class Belleza(Producto):
    def __init__(self, nombre, precio, fecha_importacion, numero_serie, es_organico, stock):
        super().__init__(nombre, precio, fecha_importacion, numero_serie, stock)
        self.es_organico = es_organico

    def get_es_organico(self):
        return self.es_organico

    def set_es_organico(self, es_organico):
        self.es_organico = es_organico
